scripts/index.py

```diff
@@ -45,9 +49,23 @@ def index():
         if 'cudl.lib.cam.ac.uk' in extlink:
             iiif_link = extlink.replace('/view/', '/iiif/')
 
+        pgpid = row['PGPID']
+        text = text_blob = None
+        if pgpid in transcriptions:
+            # TBD: should labels also be searchable ?
+            # throw everything into a text blob
+            text_blob = ' '.join(transcriptions[pgpid]['lines'])
+            # but also index as lines so highlighting can return single lines
+            # instead of the whole text
+            text = transcriptions[pgpid]['lines']
+            # index language from spreadsheet?
+            # TODO: languages will need to be auto-detected; may be useful
+            # to check against language from the spreadsheet
```

Comment:
Should we create an experiment issue to track trying this out?

Reply:
@thatbudakguy yes, good point. I've been meaning to create one so I can document the spacy library you recommended for this.